apps/web/app/api/cron/import/csv/route.ts

```diff
@@ -182,29 +188,28 @@ export async function POST(req: Request) {
                 })),
                 skipDuplicates: true,
               }),
-              domainsNotInWorkspace.flatMap((domain) =>
-                addDomainToVercel(domain),
+              // create domains in Vercel
+              domainsNotInWorkspace.map((domain) => addDomainToVercel(domain)),
+              // create links for domains
+              domainsNotInWorkspace.map((domain) =>
+                createLink({
+                  ...DEFAULT_LINK_PROPS,
+                  domain,
+                  key: "_root",
+                  url: "",
+                  tags: undefined,
+                  userId,
+                  projectId: workspace.id,
+                }),
               ),
             ]);
           }
 
           addedDomains.push(...domainsNotInWorkspace);
 
           // Process all links, including domain links
```

Comment:
Might just want to update this comment since we're not including domain links here now